packages-content-model/roosterjs-content-model-plugins/lib/tableEdit/TableEditPlugin.ts

```diff
@@ -0,0 +1,175 @@
+import normalizeRect from '../pluginUtils/Rect/normalizeRect';
+import TableEditor from './editors/TableEditor';
+import { isNodeOfType } from 'roosterjs-content-model-dom';
+import type { EditorPlugin, IEditor, PluginEvent, Rect } from 'roosterjs-content-model-types';
+
+const TABLE_RESIZER_LENGTH = 12;
+
+/**
+ * TableEdit plugin, provides the ability to resize a table by drag-and-drop
+ */
+export class TableEditPlugin implements EditorPlugin {
+    private editor: IEditor | null = null;
+    private onMouseMoveDisposer: (() => void) | null = null;
+    private tableRectMap: { table: HTMLTableElement; rect: Rect }[] | null = null;
+    private tableEditor: TableEditor | null = null;
+
+    /**
+     * Construct a new instance of TableResize plugin
+     * @param anchorContainerSelector An optional selector string to specify the container to host the plugin.
+     * The container must not be affected by transform: scale(), otherwise the position calculation will be wrong.
+     * If not specified, the plugin will be inserted in document.body
+     */
+    constructor(private anchorContainerSelector?: string) {}
```

Comment:
Just curious, are we removing the onShowHelperElement helper to allow users customize the TableEditors?

Reply:
Yes. If it is requested it will be re- added later.

Reply:
Any technical implication why we are doing this? Sounds like regressing functionality from the previous version.

Reply:
At the first step of porting a plugin, I prefer to just port the necessary functionalities. For now we don't guarantee 100% functional compatible to earlier builds. Later when need, we can add the lost functionalities back.